tests/testapp/test_locks.py

```diff
@@ -262,6 +262,7 @@ def test_write(self):
         assert Alphabet.objects.count() == 0
 
     def test_write_with_table_name(self):
+        Alphabet.objects.create(a=1)
```

Comment:
Why is this necessary? You can't lock an empty table on MySQL 8? You could put it in `setUp()` to fix both broken tests and add a comment